theme/layouts/_language.ejs

```diff
@@ -1,12 +1,12 @@
 <label>
```

Comment:
Do we need this label element?

Reply:
No, I just copied language.ejs markup. I can remove it

Reply:
Meeting notes: Narek will remove label

Reply:
I removed label from new version of custom language selector but sure about that because we need to find out why it was used in [](http://cmints.io/) ?